core/server/common/src/main/java/alluxio/master/journal/raft/JournalStateMachine.java

```diff
@@ -537,8 +537,12 @@ public synchronized void suspend(Runnable interruptCallback) throws IOException
   public synchronized void resume() throws IOException {
     LOG.info("Resuming raft state machine");
     mInterruptCallback = null;
-    mJournalApplier.resume();
-    LOG.info("Raft state machine resumed");
+    if (mJournalApplier.isSuspended()) {
+      mJournalApplier.resume();
```

Comment:
@bf8086 Can you provide a timeline for the double resume case?

Reply:
Basically the backup starts and suspends the journal, then the journal resumes, interrupts the backup and reloads from the snapshot, and then the backup attempts to resume the journal again and hit this issue.

Reply:
Got it, thanks!